test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -10118,6 +10107,80 @@ SimpleTestPackageContext createTestPackage(string name, string version, List<Sim
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        [InlineData(null)]
+        public async Task RestoreNetCore_CPVMProject_ManagePackageVersionsCentrally_CanBeDisabled(bool? managePackageVersionsCentrally)
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
```

Comment:
I'd prefer an "inline using" to avoid lots of indenting.

Reply:
There's around 100 of these in this test file.  I think I'll leave it as-is for now so its consistent with the rest of the test class and if we want to improve this later as a tech debt issue we can do it then.

Reply:
I have re-written this test from scratch, please re-review it.